### PR TITLE
nmtui, am, colordiff, dash, bastet: add Italian Translation

### PR DESCRIPTION
**Pull request by gsobell**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.it/linux/nmtui.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#
```
Comment (marchersimon) [inline pages.it/linux/nmtui.md:1]:
```suggestion
# nmtui
```

Comment (gsobell) [reply pages.it/linux/nmtui.md]: Changed, done.

**Comment by navarroaxel** [comment]
Oh, you create this PR from the `main` of your fork, and also includes the changes of the another PR. Could you fix the history of this PR? Do you need help to do that?

**Comment by gsobell** [comment]
What is the easiest way to do so?

**Comment by navarroaxel** [comment]
Well, you have 19 commits here, so:
```bash
git reset HEAD~19 --soft # undo all the new commits keeping the changes
```
Then edit the files, removing what you don't want in this PR, and commit them.
```bash
git add .
git commit ... # blah
git push --force origin main # to rewrite the history of the current branch
```

**Comment by gsobell** [comment]
@navarroaxel  Resolved the conflict with `pages.it/common/more.md` . Does PR need any more review?

**Comment by CleanMachine1** [comment]
?

**Comment by gsobell** [comment]
While attempting to to fix the commits in this PR to only include the relevant files, I reverted my remote further than intended, closing the PR. I couldn't open the PR again, so I will issue a new one once I translate a few more files. Thank you @CleanMachine1 @navarroaxel @marchersimon

**Comment by navarroaxel** [comment]
thanks to you for your time and collaboration @gsobell, I'll wait for your new submissions. :muscle:

**Comment by navarroaxel** [comment]
@gsobell remember that you never "loose" a commit in Git, you can recover old commits using `git reflog` and you can checkout them. The unused commits are cleaned when the `git gc` is runned, manually or automatically.

**Comment by marchersimon** [comment]
Any idea why @tldr-bot randomly keeps doing that?

**Pull request closed without merging**